src/loader/AMASSLoader.py

```diff
@@ -0,0 +1,279 @@
+    module is loaded first in the config file.
+
+    Example 1: generate a pose of a human kicking a ball (male model). There are different categories recorded in the mocap datasets inside of the AMASS database, for the CMU dataset we can see that the category number `10` contains multiple trials of humans kicking a ball.
+
+    {
+        "module": "loader.AMASSLoader",
+        "config": {
+        "data_path": "<args:0>",
+        "sub_dataset_id": "CMU",
+        "body_model_gender": "male",
+        "subject_id": "10",
+        "sequence_id": "1",
+        "frame_id": "600",
+      },
+    }
+
+    Example 2: generate a pose of human picking up golf ball (female model). Here we choose from the CMU dataset the subject number 64, which contains multiple trials of a human playing golf, if frame_id is not specified (by removing it from the configuration arguments), a random frame will be chosen from the motion trial.
+
+    {
+        "module": "loader.AMASSLoader",
+        "config": {
+        "data_path": "<args:0>",
+        "sub_dataset_id": "CMU",
+        "body_model_gender": "female",
+        "subject_id": "64",
+        "sequence_id": "27",
+      },
+    }
+
+    **Configuration**:
+
+    .. csv-table::
+       :header: "Parameter", "Description"
+
+       "data_path", "The path to the AMASS Dataset folder in resources folder. Type: string. Default: 'resources/AMASS'."
+       "sub_dataset_id", "Identifier for the sub dataset, the dataset which the human pose object should be extracted from. Type: srtring. Available: ['CMU', 'Transitions_mocap', 'MPI_Limits', 'SSM_synced', 'TotalCapture', 'Eyes_Japan_Dataset', 'MPI_mosh', 'MPI_HDM05', 'HumanEva', 'ACCAD', 'EKUT', 'SFU', 'KIT', 'H36M', 'TCD_handMocap', 'BML']" 
+       "body_model_gender", "The model gender, pose will represented using male, female or neutral body shape. Type: string. Available:[male, female, neutral]" 
+       "subject_id", "Type of motion from which the pose should be extracted, this is dataset dependent parameter. Type: int." 
+       "sequence_id", "Sequence id in the dataset, sequences are the motion recorded to represent certain action. Type: int." 
+       "frame_id", "Frame id in a selected motion sequence. Type: int."
+    """
+
+    # dictionary contains mocap dataset name and path to its sub folder within the main dataset, dictionary will
+    # be filled from taxonomy.json file which indicates the supported datastests
+    supported_mocap_datasets = {}
+
+    # hex values for human skin tone to sample from
+    human_skin_colors = ['2D221E', '3C2E28', '4B3932', '5A453C', '695046', '785C50', '87675A', '967264', 'A57E6E',
+                         'B48A78', 'C39582', 'D2A18C', 'E1AC96', 'F0B8A0', 'FFC3AA', 'FFCEB4', 'FFDABE', 'FFE5C8']
+
+    def __init__(self, config):
+        LoaderInterface.__init__(self, config)
+        self._data_path = Utility.resolve_path(
+            self.config.get_string("data_path", "resources/AMASS"))
```

Comment:
No `/` please, we now have community support for windows:
```suggestion
            self.config.get_string("data_path", os.path.join("resources", "AMASS")))
```